app/routes/bbl.js

```diff
@@ -0,0 +1,9 @@
+import Ember from 'ember';
+import bblDemux from '../utils/bbl-demux';
+
+export default Ember.Route.extend({
+  model(params) {
+    const bbl = bblDemux(params.bbl);
```

Comment:
You might also have said:

```javascript
export default Ember.Route.extend({
  model({ bbl }) {
    const { boro, block, lot } = bblDemux(bbl);
    this.transitionTo('lot', boro, block, lot);
  }
});
```

Reply:
oooh, better. Should have waited for your code review